sample_formatted_code.do_not_merge.c

```diff
@@ -0,0 +1,562 @@
+                              (float)ptv->last_stats64.ps_recv) *
+                              100
+                    : 0;
+}
+
+//--- enum style ---
+// clang 11: AllowShortEnumsOnASingleLine
+// clang < 11:
+// - merges short enums on one line if BraceWrapping: AfterEnum: false
+// - one-value-by-line if BraceWrapping: AfterEnum: true
+enum Gugus { bla, bli, blu };
+
+enum ThisIsTooLongForOneLine {
+    blablablablablablablabla,
+    blibliblibliblibliblibli,
+    blublublublublublublublu
+};
+
+enum { A, B } myEnum;
+
+// trailing comma forces one-value-by-line
+enum {
+    NFS_DECODER_EVENT_EMPTY_MESSAGE,
+};
+
+//--- union style ---
+typedef union {
+    int gugus;
+} Bla;
+
+union bla {
+    int gugus;
+};
+
+// --- struct style ---
+struct bla {
+    int gugus;
+};
+
+struct bla_ {
+    int gugus;
+} Bla;
+
+typedef struct bla_ {
+    int gugus;
+} Bla;
+
+//--- Alignment ---
+struct bla {
+    // AlignConsecutiveDeclarations
+    // AlignTrailingComments
+    int a;       /* comment */
+    unsigned bb; /* comment */
+    int *ccc;    /* comment */
+};
+
+// pointers
+// DerivePointerAlignment
+// PointerAlignment
+void *ptr;
+void f(int *a, const char *b);
+void (*foo)(int *);
+
+void alignment()
+{
+    // multiple consecutive vars (comments and vars can be aligned)
+    // AlignConsecutiveAssignments
+    // AlignConsecutiveDeclarations
+    // AlignTrailingComments
+    int a = 13;           /* comment */
+    int32_t abc = 1312;   /* comment */
+    int abcdefghikl = 13; /* comment */
+
+    // AlwaysBreakBeforeMultilineStrings
+    aaaa = "bbbb"
+           "ccc";
+
+    //--- variable init continuation behaviour
+    // Cpp11BracedListStyle impacts if space at beginning and end of brace
+    // e.g. false: Bla bla[] = { 1, 2, 3 };
+    //      true:  Bla bla[] = {1, 2, 3};
+    // Note, the different indentation of continuation line:
+    //  - Cpp11BracedListStyle:false does NOT use ContinuationIndentWidth (8)
+    //    but rather the regular IndentWidth (4)! Bug? Feature?
+    //  - Cpp11BracedListStyle:true uses ContinuationIndentWidth (8)
+    int list[] = {1, 2, 3, 4, 5, 6, 7, 8, 9};
+    int list[] = {
+            1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18};
+    int list[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18,
+            19, 20, 21, 22};
+    PcapStats64 last = {0, 1234, 8765};
+    // trailing comma forces one-value-by-line
+    PcapStats64 trailing_comma = {
+            0,
+            1234,
+            8765,
+    };
+    struct pcap_stat current = {12, 2345, 9876, 1345};
+    struct pcap_stat current = {
+            12, 2345, 9876, 1345, 333, 444, 5555, 66666, 7777777};
+    current = {12, 2345, 9876, 1345, 333, 444, 5555, 66666, 7777777, 888888,
+            9999999};
+
+    //--- designated initializer continuation behaviour
+    // Current clang-format disables BinPacking for designated intializers when
```

Comment:
I left this out in code-style.rst. Seemed too mundane of a detail. Only useful for initial choice of setting, imho.